From 0516c808f526f87d20a0b163fee5b937ec8e919f Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Steven!=20Ragnar=C3=B6k?= <steven@nuclearsandwich.com>
Date: Mon, 21 Mar 2022 20:35:49 -0700
Subject: [PATCH 2/2] Update python versions in PYTHONPATH patch.

---
 debian/rules.em | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/rules.em b/debian/rules.em
index 450705a..61e624d 100755
--- a/debian/rules.em
+++ b/debian/rules.em
@@ -27,7 +27,7 @@ export DEB_HOST_MULTIARCH := $(shell dpkg-architecture -qDEB_HOST_MULTIARCH)
 DEB_HOST_GNU_TYPE ?= $(shell dpkg-architecture -qDEB_HOST_GNU_TYPE)
 
 # Needed to bootstrap since the ros_workspace package does not yet exist.
-export PYTHONPATH=@(InstallationPrefix)/lib/python3.9/site-packages:@(InstallationPrefix)/lib/python3.8/site-packages
+export PYTHONPATH=@(InstallationPrefix)/lib/python3.10/site-packages:@(InstallationPrefix)/lib/python3.9/site-packages
 
 %:
 	dh $@@ -v --buildsystem=cmake --builddirectory=.obj-$(DEB_HOST_GNU_TYPE)
-- 
2.35.3

